### DISCUSSION TITLE
Player Join Whitelist?

### DISCUSSION THREAD

**Discussion by Tazaur** [vrcx-team/VRCX#690] [Ideas]
We can accept invites and requests through VRCX, so what if we just had a list that would auto accept requests from specified users? so long as we have VRCX running in the background, we those people would always be able to join off of us!

**Comment by Myrkie** [comment]
please post feature requests in the correct location [VRCX issues](https://github.com/vrcx-team/VRCX/issues) using the feature request template.

![image](https://github.com/vrcx-team/VRCX/assets/20288698/348fd881-1aa7-48c8-a3d3-e7ec72a14abb)
